packages/data/src/data-manager.ts

```diff
@@ -367,9 +369,9 @@ export class DataManager implements DataManagerInterface {
         // If there are some matched audiences
         if (
           !visitorProperties ||
-          matchedAudiences.length ||
-          matchedSegmentations.length ||
-          !audiences.length // Empty audiences list means there's no restriction for the audience
+          !audiencesToCheck.length || // Empty audiences list means there's no restriction for the audience
```

Comment:
I think in here should be `audiences.length` @abbaseya  right?

Reply:
hmmm .. I think so yes, fixing